docs/book/src/tasks/certs/experimental-ca-rotation.md

```diff
@@ -0,0 +1,103 @@
+## Experimental CA rotation procedure
+
+<aside class="note warning">
+
+<h1>Cluster API CA rotation support</h1>
+
+CA rotation is not supported (nor automated) by Cluster API, please use this procedure at your own risk!
+Rotating CA is a tough Kubernetes topic on its own, it's even harder when everything is wrapped around Cluster API automation and immutability concepts.
+
+An issue is currently tracking discussions around this feature request : [Automatic CA rotation in CAPI #7721](https://github.com/kubernetes-sigs/cluster-api/issues/7721). Reading the issue is strongly advised to really be sure of what you are going to do.
+
+The recommended strategy in CAPI is to destroy and recreate clusters ahead of the expirty date to avoid CA expiry.
```

Comment:
```suggestion
Rotating CA is a tough Kubernetes topic on its own, and CA rotation is currently not supported (nor automated) by Cluster API; however, we are providing this guide as a reference in case you want to experiment with this procedure. 

We highly recommend testing this procedure in non-prod environments first and reporting back feedback.

An issue is currently tracking discussions around implementing this feature in Cluster API : [Automatic CA rotation in CAPI #7721](https://github.com/kubernetes-sigs/cluster-api/issues/7721). Reading the issue is strongly advised to really be sure of what you are going to do.

As an alternative, you have to migrate your workloads to newly created clusters ahead of the expiry date to avoid CA expiry (10y).
```

just rephrasing a little bit
cc @vincepri @CecileRobertMichon